### ISSUE TITLE
Troubleshoot why coveralls broke

### ISSUE THREAD

**Issue by junkert** [smithoss/gonymizer#90]
I removed this for our 2.0 release to get the release through, but removed Coveralls in the process. We should add this or Codecov back to the project

**Comment by wavemoran** [comment]
Yeah, we moved everything back to Codecov internally. Seems they fixed their Github notification issues and it's worked a lot better for us.
